tests/dragonfly/replication_test.py

```diff
@@ -1993,6 +1993,9 @@ async def send_setex():
     await wait_available_async(c_replica)
     await stream_task
 
-    # TODO: compare master and replica once we have the utility to do so.
+    await asyncio.sleep(1.0)
```

Comment:
Instead use await check_all_replicas_finished([c_replica], c_master)

Reply:
sorry, I missed this comment